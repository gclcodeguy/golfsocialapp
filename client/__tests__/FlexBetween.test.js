import React from "react";
import renderer from "react-test-renderer";
import FlexBetween from "../src/components/FlexBetween.jsx";
//unit test for flexBetween component
describe("FlexBetween", () => {
  it("should render correctly", () => {
    const tree = renderer
      .create(<FlexBetween>Hello World</FlexBetween>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
