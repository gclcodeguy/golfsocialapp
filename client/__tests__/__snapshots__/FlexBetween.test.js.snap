// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexBetween should render correctly 1`] = `
<div
  className="MuiBox-root css-1omog99"
>
  Hello World
</div>
`;
